packages/browserify/test/generateConfig.js

```diff
@@ -83,11 +87,12 @@ test('generateConfig - policy ignores global refs when properties are not access
       typeof window !== undefined
     },
     defaultConfig: false,
+    expectedResult: {
+      resources: {}
+    }
   })
   const policy = await autoConfigForScenario({ scenario })
-  t.deepEqual(policy, {
-    resources: {}
-  }, 'policy matches expected')
+  scenario.checkResult(t, policy, scenario)
```

Comment:
dont use checkResult to assert an unrelated value 
just keep the original assertion